lib/lotus/view/configuration.rb

```diff
@@ -388,6 +409,7 @@ def reset!
         root             DEFAULT_ROOT
         default_encoding DEFAULT_ENCODING
 
+        @partials   = Hash.new
```

Comment:
@stevehook Can you please add unit tests for `Configuration`?

See:
  * https://github.com/lotus/view/blob/master/test/view/configuration_test.rb#L128
  * https://github.com/lotus/view/blob/master/test/view/configuration_test.rb#L191
  * https://github.com/lotus/view/blob/master/test/view/configuration_test.rb#L295
  * https://github.com/lotus/view/blob/master/test/view/configuration_test.rb#L331

:+1: